src/components/homepage/Homepage.jsx

```diff
@@ -1,9 +1,10 @@
+import Movie from "../movie/Movie";
+import { MovieContainer } from "./Styles/StyleMovieContainer";
+import { GetMovies } from "../../API/movies";
+
 const Homepage = () => {
-    return (
-        <div className="homepage">
-            <p>This is home page</p>
-        </div>
-    );
-}
+  const { movies, isPending, error } = GetMovies();
+  return <MovieContainer>{movies && <Movie movies={movies} />}</MovieContainer>;
```

Comment:
```suggestion
  return (
  <MovieContainer>
     {movies && <Movie movies={movies} />}
  </MovieContainer>);
```

Reply:
Proper formatting is important because it makes the code more readable.

Reply:
Done